src/main/java/de/mediathekview/mserver/crawler/sr/tasks/SrFilmDetailTask.java

```diff
@@ -0,0 +1,213 @@
+package de.mediathekview.mserver.crawler.sr.tasks;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mserver.base.utils.DateUtils;
+import de.mediathekview.mserver.base.utils.HtmlDocumentUtils;
+import de.mediathekview.mserver.crawler.ard.json.ArdVideoInfoDTO;
+import de.mediathekview.mserver.crawler.ard.json.ArdVideoInfoJsonDeserializer;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.sr.SrTopicUrlDTO;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.CrawlerTool;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Node;
+import org.jsoup.select.Elements;
+
+public class SrFilmDetailTask extends AbstractDocumentTask<Film, SrTopicUrlDTO> {
+  
+  private static final org.apache.logging.log4j.Logger LOG = LogManager.getLogger(SrFilmDetailTask.class);
+
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+    DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm", Locale.GERMANY);
+  private static final DateTimeFormatter DATE_TIME_FORMATTER_ENG =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm", Locale.GERMANY);
+
+  private static final String CONTENT_SELECTOR = "div.article__content";
+  private static final String TITLE_SELECTOR = CONTENT_SELECTOR + " > div > h3";
+  private static final String DETAILS_SELECTOR = CONTENT_SELECTOR + " > div > p";
+  private static final String DESCRIPTION_SELECTOR = CONTENT_SELECTOR + " > h1:contains(Themen)";
+  private static final String VIDEO_DETAIL_ATTRIBUTE = "data-mediacollection-ardplayer";
+  private static final String VIDEO_DETAIL_SELECTOR = "div[" + VIDEO_DETAIL_ATTRIBUTE + "]";
+  
+  public SrFilmDetailTask(final AbstractCrawler aCrawler,
+      final ConcurrentLinkedQueue<SrTopicUrlDTO> aUrlToCrawlDTOs) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)